app/models/tag_mapping.rb

```diff
@@ -0,0 +1,6 @@
+class TagMapping < ActiveRecord::Base
```

Comment:
Wondering about the naming. Maybe `BulkTagMapping` or something? Or maybe we could put it all in a namespace?